components/resources/ResourceInformation.js

```diff
@@ -16,20 +17,29 @@ function ResourceInformation({ resourceId }) {
     error,
   } = useOperation("getResource", resourceId);
 
+  const [average, setAverage] = useState(undefined);
```

Comment:
There is too much logic in this component. I really tried to join these with Ronny branch but in the end was better to start from the begining.